backend/src/main/java/cz/cuni/mff/xrg/odcs/backend/execution/pipeline/Executor.java

```diff
@@ -136,21 +135,12 @@
      */
     private String backendID;
 
-    private boolean clusterMode = false;
-
     /**
      * Sort pre/post executors.
      */
     @PostConstruct
     public void init() {
-        try {
-            this.clusterMode = this.appConfig.getBoolean(ConfigProperty.BACKEND_CLUSTER_MODE);
-        } catch (MissingConfigPropertyException e) {
-            LOG.info("Running in single mode because cluster mode property is missing in config.properties, {}", e.getLocalizedMessage());
-        }
-        if (this.clusterMode) {
-            this.backendID = this.appConfig.getString(ConfigProperty.BACKEND_ID);
-        }
+        this.backendID = this.appConfig.getString(ConfigProperty.BACKEND_ID);
```

Comment:
Backend process will not even start without this property and error messages is shown in logs:
`Invocation of init method failed; nested exception is cz.cuni.mff.xrg.odcs.commons.app.conf.MissingConfigPropertyException: Config is missing property: backend.id.`

This is the standard behavior for mandatory parameters after error handling in backend was fixed